services/distribution/src/main/resources/application.yaml

```diff
@@ -95,6 +97,9 @@ services:
       min-ios: ${MIN_IOS_VERSION:0.5.0}
       latest-android: ${LATEST_IOS_VERSION:1.0.4}
       min-android: ${MIN_ANDROID_VERSION:1.0.4}
+      # With ENF v2 Android apps are versioned by Version Code and not by Semantic Versioning
+      min-android-version-code: ${ANDROID_MIN_VERSION_CODE:31}
+      latest-android-version-code: ${ANDROID_LATEST_VERSION_CODE:31}
```

Comment:
The iOS version information for the v2 App Config format is missing:
![image](https://user-images.githubusercontent.com/2060790/98667103-43644c80-234e-11eb-9fa6-e01e2bdabcdc.png)

Reply:
Can't the old environment variables be used for this purpose?   
      latest-ios: ${LATEST_IOS_VERSION:0.8.2}
      min-ios: ${MIN_IOS_VERSION:0.5.0}

Reply:
Can we change this for v1 and v2 App Config to `IOS_MIN_VERSION` / `IOS_LATEST_VERSION` so that it's consistent with the other parameter names?